GoogleForCreators/web-stories-wp
# Prevent click event propagation from resize handle

## Context
Click event propagated from the resize handle in a resizeable panel will trigger `onClick` in the panel header which results in collapsing the panel, as in case of the Layer Panel for example.

This PR will prevent click propagation from the resize handle since this should only handle dragging and double clicks.


## Summary



## Relevant Technical Choices
An alternative could be rewriting the panel header so that it would not be a button and instead, the panel header would consist of wrapper, resize handle (if resizeable), and the clickable header part. Just stopping the propagation is a simpler solution, however, and seems to fill the purpose in this case.


## To-do



## User-facing changes



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Add some elements
2. Try resizing the layer panel
3. Verify it's working as expected

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6122
